libraries/core-react/stories/components/Tooltip.stories.tsx

```diff
@@ -53,14 +53,16 @@ export const Default: Story<TooltipProps> = (args) => {
         aria-describedby="tooltip"
         onMouseEnter={handleOpen}
         onMouseLeave={handleClose}
+        onFocus={handleOpen}
+        onBlur={handleClose}
       >
         Hover me!
       </Button>
       <Tooltip
-        {...args}
         open={openState}
         id="tooltip"
         anchorEl={referenceElement.current}
+        {...args}
```

Comment:
🙈

Reply:
Haha played with some things to see if it could fix the weird code preview in Storybook